From f73cdeac19774c7786ae71e0b38de9b1df39ea98 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michel=20D=C3=A4nzer?= <mdaenzer@redhat.com>
Date: Tue, 14 Jan 2020 09:23:34 +0100
Subject: [PATCH] modesetting: Explicitly #include "mi.h"

For the miClearDrawable prototype. Apparently it doesn't get pulled in
for some build configurations, breaking the build.

Downloaded from upstream PR:
https://gitlab.freedesktop.org/xorg/xserver/merge_requests/380

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 hw/xfree86/drivers/modesetting/drmmode_display.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/xfree86/drivers/modesetting/drmmode_display.c b/hw/xfree86/drivers/modesetting/drmmode_display.c
index b826bee6a..d13cec077 100644
--- a/hw/xfree86/drivers/modesetting/drmmode_display.c
+++ b/hw/xfree86/drivers/modesetting/drmmode_display.c
@@ -37,6 +37,7 @@
 #include "inputstr.h"
 #include "xf86str.h"
 #include "X11/Xatom.h"
+#include "mi.h"
 #include "micmap.h"
 #include "xf86cmap.h"
 #include "xf86DDC.h"
-- 
2.24.1

